common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -952,6 +954,14 @@ def update_score(self, data):
         self.lcp.update_score(score_msg, queuekey)
         self.set_state_from_lcp()
         self.set_score(self.score_from_lcp())
+        if self.course_id == CourseLocator.from_string('course-v1:MITx+6.00.1x+2T2017_2'):
```

Comment:
More simple if `unicode(self.course_id)=='course-v1:MITx+6.00.1x+2T2017_2'` but as this is temporarly logging, this looks good to me.